taraxa/state/rewards_stats/rewards_stats.go

```diff
@@ -21,8 +21,14 @@ type RewardsStats struct {
 	// Total unique transactions counter
 	TotalUniqueTxsCount uint32
 
-	// Total unique votes counter
-	TotalUniqueVotesCount uint32
+	// Total weight of votes in block
+	TotalVotesWeight uint64
+
+	// Bonus weight of votes in block
+	BonusVotesWeight uint64
+
+	// Max weight of votes in block
+	MaxVotesWeight uint64
```

Comment:
I would also add panic check TotalVotesWeight + BonusVotesWeight <= MaxVotesWeight into the ```
taraxa_evm_state_api_transition_state. There is similar validation check:
	disabled_contract_distribution := st.GetChainConfig().BlockRewardsOptions.DisableContractDistribution
	if !disabled_contract_distribution && len(params.Txs) != len(params.TxsValidators) {
		errorString := fmt.Sprintf("Number of txs (%d) != number of txs validators (%d)", len(params.Txs), len(params.TxsValidators))
		panic(errorString)
	}
```